docs/how-to-deploy-cp4waiops.md

```diff
@@ -2,7 +2,7 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-- [Deploy CP4WAIOps 3.3 using GitOps](#deploy-cp4waiops-33-using-gitops)
+- [Deploy CP4WAIOps 3.4 using GitOps](#deploy-cp4waiops-33-using-gitops)
```

Comment:
Instead of manually update it, the inline links need to be auto-regenerated using doctoc.